furusiya/view/data/saveload.py

```diff
@@ -0,0 +1,28 @@
+import shelve
+
+
+class SaveLoad:
```

Comment:
This shouldn't be in the `view` directory. It should be under some sort of `i/o` type of directory (but not `io` as we've seen with package/namespace shadowing).